services/modules/slack/manager.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import unicode_literals
+import random
+import string
+import requests
+from eveonline.managers import EveManager
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from slackclient import SlackClient
+
+from six import iteritems
+from .tasks import SlackTasks
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SlackManager:
+    def __init__(self):
+        pass
+
+    @classmethod
+    def _response_ok(cls, response):
+        try:
+            if response['ok']:
+                return True
+            else:
+                return False
+        except:
+            return False
+
+    @staticmethod
+    def exec_request(endpoint, func, email, **kwargs):
+        try:
+            sc = SlackClient(settings.SLACK_TOKEN)
+            res = sc.api_call('users.admin.invite', email=email, resend=True)
+            return res
+        except:
```

Comment:
I really don't like these massive catch alls in all the service managers. At the very least if you insist on using it, raise another exception or something for the caller to handle rather than these empty dicts. I know this isn't the first time this has appeared in service managers, but I'd like to avoid adding _even more_ of them.